lib/DnD/Draggable.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { func, string, node } from 'prop-types';
+import { DragPreviewImage, useDrag } from 'react-dnd';
+import { getEmptyImage } from 'react-dnd-html5-backend';
+import { DndContext } from './DndProvider';
+
+function Draggable({ children, type, preview }) {
+  const { setIsDragging, setPreview } = useContext(DndContext);
+  const [collect, defineDragRef, definePreviewRef] = useDrag({
+    item: { type },
+    begin: () => {
+      setPreview(preview);
+      setIsDragging(true);
+    },
+    end: () => setIsDragging(false),
+    collect: monitor => ({
+      itemType: monitor.getItemType(),
+      isDragging: !!monitor.isDragging()
+    })
+  });
+
+  return (
+    <>
+      {preview && (
+        <DragPreviewImage
```

Comment:
This disables the default drag preview.

It took me some time to figure it out but it turns out you can only set an image to replace the default preview. However, if you set it to an empty image then the preview is disabled (I saw we were doing this in the app 👍 )

Reply:
Ha yeah, this is a little weird quirk